pkg/api/api.go

```diff
@@ -26,6 +26,7 @@ func (hs *HTTPServer) registerRoutes() {
 	r := hs.RouteRegister
 
 	// not logged in views
+	r.Get("/", reqSignedIn, hs.Index)
```

Comment:
Doesn't `reqSignedIn` require the client to be signed in? That contradicts "not logged in views", right? Why is this change necessary?

Reply:
Was reverting it back to a previous state (see https://github.com/grafana/grafana/blob/7b7b95341e73d0ac173e9ccef25cc129048b8701/pkg/api/api.go#L27) but I agree that it's a bit confusing

Reply:
Seems like you're reverting #19118. Maybe need to verify with Dominik this change is intended/works as expected.

Reply:
Is the idea that #19118 can be reverted, so long as you make the change on line 291 of pkg/api/http_server.go?

@dprokop What do you think?

Reply:
I think this looks fine as the not found handler now has req signed in check

Reply:
Yeah, think this is ok to revert that change. Let me double-check if we are not introducing the same bug again.

Reply:
If we are reverting the change, can we move the route handler (`r.Get("/", reqSignedIn, hs.Index)`) outside the "not logged in views" section? It doesn't sound right to me.

Reply:
Checked - all good!